src/pytest_html/nextgen.py

```diff
@@ -66,11 +66,19 @@ def __init__(self, title, config):
                 "collectedItems": 0,
                 "runningState": "not_started",
                 "environment": {},
-                "tests": [],
+                "tests": defaultdict(list),
```

Comment:
Made a small performance improvement to avoid having to iterate through all tests, we key them with the original `nodeid`.